packages/plugins/IconMenu/README.md

```diff
@@ -11,40 +11,70 @@ _Please note that the average user has never seen the client before and just wan
 
 ## Configuration
 
+### iconMenu
+
 > ⚠️ This plugin can't be configured with the `mapConfiguration`, but is configured during client construction. Currently, no way exists to add plugins to it after the build took place.
 
 For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
 
-Usage during client build:
+| fieldName | type | description |
+| - | - | - |
+| initiallyOpen  | string?   | Id of the plugin which should be open on start; only applicable if the device doesn't have a small display  |
+| menus | Array | Defines which plugins should be rendered as part of the icon menu. |
+| layoutTag | enum NineLayoutTag | Defines where the icon menu is rendered. Only use `NineLayoutTag.TOP_RIGHT` here for the time being. |
```

Comment:
https://github.com/Dataport/polar/wiki/Architecture-Decision-Records-%F0%9F%93%83#2024-08-05-configuration-parameters-in-tables-have-to-be-ordered-by-a-required-and-b-alphabetically

There are some new rules for sorting parameters now.

Reply:
f536941285d77b6098b3d622547a8a24d77d1db0